.github/workflows/ci-lint-test.yml

```
@@ -31,4 +31,7 @@ jobs:
           submodules: recursive
 
       - name: Verify Protobuf types are up to date
-        run: make proto && git diff --name-status --exit-code
\ No newline at end of file
+        run: make proto && git diff --name-status --exit-code
```

guard against new files too
```suggestion
        run: make proto && { if git status --porcelain | grep '??'; then exit 1; else git diff --name-status --exit-code; fi }
```